src/Frontend/Utils.ts

```
@@ -7,16 +7,33 @@ import {
 import type { AppConfig } from "../Config/Layer.ts";
 import type { S3Client } from "../Backends/S3/Client.ts";
+import type { SwiftClient } from "../Backends/Swift/Client.ts";
 import { BadGateway } from "./Api.ts";
 
+/**
+ * Fixes header values that might have been incorrectly decoded as Latin-1
+ * instead of UTF-8 by the HTTP server.
+ */
+export function fixHeaderEncoding(value: string): string {
+  // deno-lint-ignore no-control-regex
+  if (!/[^\x00-\x7F]/.test(value)) {
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)